packages/@ember/-internals/glimmer/tests/integration/components/input-curly-test.js

```diff
@@ -266,18 +285,24 @@ moduleFor(
       // this.assertAttr('tabindex', '30'); //NOTE: failing in IE (TEST_SUITE=sauce)
     }
 
-    ['@test static attributes']() {
-      this.render(`
-      {{input type="text"
-        disabled=true
-        value="Original value"
-        placeholder="Original placeholder"
-        name="original-name"
-        maxlength=10
-        minlength=5
-        size=20
-        tabindex=30
-      }}`);
+    ['@test [DEPRECATED] static attributes']() {
+      maybeExpectDeprecation(
+        EMBER_MODERNIZED_BUILT_IN_COMPONENTS,
+        () => {
+          this.render(`
+          {{input type="text"
+            disabled=true
+            value="Original value"
+            placeholder="Original placeholder"
+            name="original-name"
+            maxlength=10
+            minlength=5
+            size=20
+            tabindex=30
+          }}`);
```

Comment:
nit: this indentation style looks pretty good to me, let's apply this to the other test cases you touched since the auto-formatter made some of them look pretty strange.